GNUmakefile

```diff
@@ -183,7 +183,7 @@ lint-deps: ## Install linter dependencies
 	@echo "==> Updating linter dependencies..."
 	GO111MODULE=on cd tools && go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.24.0
 	GO111MODULE=on cd tools && go get github.com/client9/misspell/cmd/misspell@v0.3.4
-	GO111MODULE=on cd tools && go get github.com/hashicorp/go-hclog/hclogvet@master 
+	GO111MODULE=on cd tools && go get github.com/hashicorp/go-hclog/hclogvet@4a9e55fb9e7db75382c419e20dd1eb26821ec85e
```

Comment:
Instead of the sha, use the tag, e.g.
```
github.com/hashicorp/go-hclog/hclogvet@v0.1.3
```
Since `hclogvet` is a submodule, the relevant tags are in the form `hclogvet/vX.X.X` on https://github.com/hashicorp/go-hclog/tags
And also viewable on https://pkg.go.dev/github.com/hashicorp/go-hclog/hclogvet?tab=versions

Reply:
:exploding_head: